web/themes/custom/novel/templates/views/node--event-list-teaser.html.twig

```diff
@@ -0,0 +1,31 @@
+<a href="{{ content.event_link }}" class="event-list-item arrow__hover--right-small">
+  <div class="event-list-item__image-container">
+    {% if content.field_event_image['#items'] is not empty %}
+      {% set image_uri = content.field_event_image['#items'].entity.field_media_image.entity.fileuri %}
+      {% set image_url = file_url(image_uri) %}
+        <img src="{{ image_url }}" alt="{{ content.label }}" class="event-list-item__image" />
+    {% endif %}
+  </div>
```

Comment:
Please reconsider this construct.

I think I see what you want to do but I think this belongs in the template for the media or file entity if possible.

We need to get the right classes into the template. That might be tricky. Perhaps @rasben can help here or we can look at it together.

One problem which seems to be a side effect of the current approach is that you use the title of the event as the `alt` text instead of the alt text for the media entered by the editor.

Reply:
Do we need to render the identity color box if there is no media?

Reply:
I am not sure how to accomplish it. 

I thought i could overwrite the media or field when used in a specific view mode by appplying the view mode to the end of the template --list-teaser.html.twig, but i haven't been able to get this working. 

Not sure if view mode specific overrides are not possible by defualt or if they need to be added by preprocess. I tried this also but it didn't work fully. Could also be that i need to create a view mode for the media type, but this seems strange to me. 


I largely kept the same, but added correct alt text. see more in the commit.

Reply:
in terms of whether we need to render the identity color box, i would assume so. 

I guess this might not ahve been a part of the original ticket, but since media library was added etc, and i figured the identity should be there aswell.